mirar/pipelines/sedmv2/load_sedmv2_image.py

```diff
@@ -93,36 +93,77 @@ def clean_science_header(  # pylint: disable=too-many-branches
         if isinstance(informative_hdr["QCOMMENT"], str):
             header["OBJECTID"] = informative_hdr["QCOMMENT"].split("_")[0]
 
-    if not EXPTIME_KEY in informative_hdr:
-        header[EXPTIME_KEY] = informative_hdr[
-            "EXPOSURE"
-        ]  # be weary - exposure is not always reliable
+    if not is_mode0:
+        frame_time = 10  # pretty sure this is always true
+        # HUGE CHANGE - MEF is split up, so we need individual exptimes
+        if EXPTIME_KEY not in informative_hdr:
+            # be weary - keyword "EXPOSURE" is not always reliable
```

Comment:
be wary? though SEDmV2 does make you weary...

Reply:
haha